src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -55,8 +55,8 @@ public HttpResponseMessage Invoke(string name, [FromBody] FunctionInvocation inv
         }
 
         [HttpGet]
-        [Route("admin/functions/{name}")]
-        public FunctionStatus Get(string name)
+        [Route("admin/functions/{name}/status")]
+        public FunctionStatus GetFunctionStatus(string name)
         {
             FunctionStatus status = new FunctionStatus();
             Collection<string> functionErrors = null;
```

Comment:
I don't think there is an issue. The function status route is guarded by the EnsureHostRunningHandler which won't allow requests through if the host isn't running